### PR TITLE
Adds banner and removes making us look bad

### PR DESCRIPTION
**Pull request by JamieD1**
Adds new banner and removes the dynamic lag count as it makes us look bad.

#### Changelog

:cl:  
rscadd: Added Banner
rscdel: Removed dynamic lag counter
/:cl:

### CODE REVIEW

File: yogstation/code/game/world.dm [hunk 1]
```diff
@@ -41,13 +41,12 @@ GLOBAL_LIST_EMPTY(donators)
 	var/server_name = CONFIG_GET(string/servername)
 	if (server_name)
 		s += "<b>[server_name]</b> &#8212; "
-	
-	s += "<b>[station_name()]] -- [round(10000 / (100 + SStime_track.time_dilation_current), 1)]% LAG FREE</b><br>"; // The station & server name line
+
+	s += "<b>[station_name()]] -- 99% LAG FREE</b><br>"; // The station & server name line
```
Comment (jcatjmeow) [inline yogstation/code/game/world.dm]:
```suggestion
	s += "<b>[station_name()]] -- 99% FREE LAG</b><br>"; // The station & server name line
```

**Review by jcatjmeow — CHANGES_REQUESTED** [review]

**Review by alexkar598 — DISMISSED** [review]
Banner wont display, we ran out of characters for our description q long time ago. We cant even display the Hosted by part

**Comment by Reed0506** [comment]
lag percentage is funny jamie ur bad

**Review by fluffe9911 — APPROVED** [review]
he did win the competition and we got to include it even if it costs being honest

**Pull request merged**